services/tables/src/main/java/com/linkedin/openhouse/tables/config/TblPropsToggleRegistry.java

```diff
@@ -0,0 +1,31 @@
+package com.linkedin.openhouse.tables.config;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+/** A central registry for all table properties that are managed through Feature Toggle. */
+@Component
+public class TblPropsToggleRegistry {
+
+  public static final String ENABLE_TBLTYPE = "enable_tabletype";
+
+  // TODO: Using these vocabularies as MySQL validation
+  private final Map<String, String> featureKeys = new HashMap<>();
```

Comment:
add a note in documentation that ** only allowlisting is supported **

Reply:
I think the documentation should be added in either the annotation or `featureToggleAspect` right? 
This registry can be used for other things, like "disallow" style feature